### PR TITLE
Support for curved lines

### PR DESCRIPTION
**Pull request by menion**
Lines, mainly contours etc. suffer from straight lines, as seen on the screenshot.
image

This PR brings support for the new attribute "curve" for lines. With the value "cubic" the result is following
image

If there are missing some updates in docs, please let me know, thanks.

### CODE REVIEW

File: resources/renderTheme.xsd [hunk 1]
```diff
@@ -188,6 +195,7 @@
         <xs:attribute name="stroke-dasharray" type="tns:strokeDasharray" use="optional" />
         <xs:attribute name="stroke-linecap" default="round" type="tns:cap" use="optional" />
         <xs:attribute name="stroke-linejoin" default="round" type="tns:linejoin" use="optional" />
+        <xs:attribute name="curve" type="xs:string" use="optional" />
```
Comment (devemux86) [inline resources/renderTheme.xsd:198]: We probably need to add `default="no"`.

**Comment by devemux86** [comment]
Thanks, nice work!

How is the performance when is enabled?

**Comment by menion** [comment]
I'm testing it now by simply changing the default from "no" to "cubic". This, in my favorite zoom level 12 near Prague, means a huge amount of ways. All will curve=cubic causes a reduction in the draw by 6%. Nothing serious I believe. It may be interesting to test it in the Alps with a lot of contour lines visible. There may be a noticeable impact I believe.

**Comment by devemux86** [comment]
@menion thanks!

**Pull request merged**